app.hypermedia.testing.dsl/src/main/java/app/hypermedia/testing/dsl/Hydra.xtext

```diff
@@ -18,15 +18,29 @@ RepresentationStep:
 	OperationBlock
 ;
 
+@Override 
+Identifier: 
+    UriName | 
+    PrefixedName
+;
+
+NamespaceDeclaration:
+    'PREFIX' prefix=PrefixDeclaration ':' namespace=URI
+;
+
+PrefixDeclaration: value=ID;
```

Comment:
@mchlrch @nicky508  please help, I'm stuck

If you look at the test results, you will see that the prefixes are not parsed. I think that the `PrefixDeclaration` declaration is incorrectly matched to the `FIELD_NAME` terminal and it gives errors like 

```
XtextSyntaxDiagnostic: null:1 mismatched input 'foaf' expecting RULE_ID
```

How do I disambiguate the language?